### DISCUSSION TITLE
Can Slack mobile app like Input Textbox be made with react-native-bottom-sheet ?

### DISCUSSION THREAD

**Discussion by saumya66** [gorhom/react-native-bottom-sheet#1327] [Q&A]
Basically, a normal-looking input turns into a draggable input like react-native-bottom-sheet when it's focused or tapped on. And then the user can drag it up if she wants to snap to full screen.

**Comment by Karthik-B-06** [comment]
I have been trying to have an expanding Text Input inside `BottomSheet`. My Text Input is not working properly because the Sheet should have `enableDynamicSizing`. Were you able to make this work?

**Reply by Vishnu-vashishth** [reply]
Hey friend, did you manage to get this working?

**Reply by Karthik-B-06** [reply]
Actually I was trying something like this with bottom sheet, I want the container to expand as per bottom sheet. Could not make it work! 


https://github.com/user-attachments/assets/1a9d2542-4cb6-4ee9-bc39-d6027a4902f6
